app/controllers/albums.js

```diff
@@ -0,0 +1,20 @@
+const albumsService = require('../services/album');
+const errors = require('../errors');
+
+module.exports.albums = () => {
+  albumsService
+    .albums()
+    .then(albumsList => albumsList)
+    .catch(() => {
+      throw errors.externalApiError('Error in external API');
```

Comment:
You are not forwarding any error to the errors middleware here. If an error occurs then this endpoint will not respond.